app/javascript.js

```diff
@@ -0,0 +1,135 @@
+  " ACME DIVISION ALL RIGHTS RESERVED";
+}
+
+//go to top and focus in sign up form
+function goTop(){
+  document.body.scrollTop = 0;
+  document.getElementById("your_name").focus();
+}
+
+//converting seconds to minutes
+function convert(time){
+  var minutes = "" + Math.floor(time / 60);
+  var seconds = "0" + (time - minutes * 60);
+  return minutes.substr(-2) + ":" + seconds.substr(-2);
+}
+
+function explorePlaylist(playlistNumber){
+  document.getElementById('playlist').className = "playlist-expand";
+  var playlist = new Playlist(playlists[playlistNumber]);
+  console.log(ModelFactory.create("Playlist", playlists[playlistNumber]));
+  var song = new Song(playlists[playlistNumber].songs[playlistNumber]);
+  song.attach(playlist);
+  song.getSongTitle();
+  var songs = playlist.getPlaylistSongs();
+  for (var i = 0; i < songs.length; i++){
+    var table = document.getElementById('playlist-songs-body');
+    var row = document.createElement('tr');
+    var col1 = document.createElement('td');
+    var col2 = document.createElement('td');
+    var col3 = document.createElement('td');
+    var col4 = document.createElement('td');
+    var img = document.createElement('img');
+
+    row.setAttribute('class','newRow');
+    img.src = songs[i].image;
+    col2.innerHTML = songs[i].songTitle + "</br>" + "by " + songs[i].songAuthor;
+    col3.innerHTML = convert(songs[i].songLength);
+    col4.innerHTML = songs[i].songListened;
+    table.appendChild(row);
+    row.appendChild(col1);
+    row.appendChild(col2);
+    row.appendChild(col3);
+    row.appendChild(col4);
+    col1.appendChild(img);
+  }
+}
+
+function closePlaylist(){
+  document.getElementById('playlist').classList.remove("playlist-expand");
+  var tableBody = document.getElementById('playlist-songs-body');
+  tableBody.innerHTML = '';
+}
+
+function getFocus(input){
+  input.style.backgroundColor = "rgba(0,191,255,0.1)";
+  input.style.borderColor = "rgb(0,191,255)";
+  input.style.borderStyle = "solid";
+  input.style.borderWidth = "1px";
+}
+
+function getBlur(input){
+  input.style.backgroundColor = "rgba(128,128,128 ,0.2)";
+  input.style.borderWidth = "1px";
+  input.style.borderColor = "rgba(128,128,128 ,0.2)";
+}
+
+function validateUser(){
+  var bool = false;
+  var email = document.getElementById('email').value;
+  var password = document.getElementById('password').value;
+  var account = new AccountService(users);
+  for(var i = 0; i < users.length; i++){
+    if(users[i].email == email && users[i].password == password){
+      account.login();
+
+    }else{
+
+      account.displayLoginError();
+      bool = false;
+    }
+    return bool;
+  }
+}
+
+//creating cards in home page
+function createCards(){
+  for (var i = 0; i < playlists.length; i++){
+    var cardDiv = document.createElement('div');
+    var matDiv = document.createElement('div');
+    var button = document.createElement('button');
```

Comment:
This is just to big. Split it in smaller methods like drawExploreBtn(), drawPlayIcon() etc